### PR TITLE
Resurrect ClickHouse fuzzing 

### PR DESCRIPTION
**Pull request by nikitamikhaylov**
Continuation of #3800

### CODE REVIEW

**Comment by nikitamikhaylov** [comment]
@googlebot I signed it!

**Comment by kyprizel** [comment]
@googlebot I consent

File: projects/clickhouse/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Copyright 2016 Google Inc.
```
Comment (jonathanmetzman) [inline projects/clickhouse/Dockerfile]: nit: 2020 Google LLC

File: projects/clickhouse/build.sh [hunk 2]
```diff
@@ -0,0 +1,111 @@
+#!/bin/bash -eu
+# Copyright 2016 Google Inc.
```
Comment (jonathanmetzman) [inline projects/clickhouse/build.sh]: Same nit

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Comment by nikitamikhaylov** [comment]
@jonathanmetzman But it may not build currently. May I continue in other PR if any issues occurs? And not revert this PR?

**Comment by jonathanmetzman** [comment]
Ooops. Sorry didn't know this. Ummm...I'll need to disable this project because new projects that fail to build cause exceptions.
When you want to fix it just set it to enabled in project.yaml.

**Pull request merged**